src/components/Collections/Collections.js

```diff
@@ -1,19 +1,37 @@
 import React from "react";
 import axios from "axios";
+import LoadingBar from "react-top-loading-bar";
+import {
+  Container,
+  StyledDiv,
+  Title,
+  Description,
+  Total,
+  PreviewPhotos,
+  Preview,
+} from "./collections.styles";
+import unavilableCover from "../../assets/UnavailableCover.png";
```

Comment:
`import unavilableCover from "assets/UnavailableCover.png";`

Reply:
yea and the last thing i import is styles, i usually import images before than styles...